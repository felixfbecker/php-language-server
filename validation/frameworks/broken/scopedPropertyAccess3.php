<?php

class A {
    static $a = "hello";
}
echo A::$a;

$a = new