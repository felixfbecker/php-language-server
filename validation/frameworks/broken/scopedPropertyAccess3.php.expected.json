{
    "references": {
        "A": [
            "\/Users\/roblou\/code\/php-language-server\/tests\/Validation\/..\/..\/validation\/frameworks\/broken\/scopedPropertyAccess3.php"
        ],
        "A::$a": [
            "\/Users\/roblou\/code\/php-language-server\/tests\/Validation\/..\/..\/validation\/frameworks\/broken\/scopedPropertyAccess3.php"
        ]
    },
    "definitions": {
        "A": {
            "extends": [],
            "isGlobal": true,
            "isStatic": false,
            "canBeInstantiated": true,
            "symbolInformation": {
                "name": "A",
                "kind": 5,
                "location": {
                    "uri": "\/Users\/roblou\/code\/php-language-server\/tests\/Validation\/..\/..\/validation\/frameworks\/broken\/scopedPropertyAccess3.php"
                },
                "containerName": ""
            },
            "type__class": "LanguageServer\\Tests\\ValidationTest",
            "type": null,
            "documentation": null
        },
        "A::$a": {
            "extends": [],
            "isGlobal": false,
            "isStatic": true,
            "canBeInstantiated": false,
            "symbolInformation": {
                "name": "a",
                "kind": 7,
                "location": {
                    "uri": "\/Users\/roblou\/code\/php-language-server\/tests\/Validation\/..\/..\/validation\/frameworks\/broken\/scopedPropertyAccess3.php"
                },
                "containerName": "A"
            },
            "type__class": "phpDocumentor\\Reflection\\Types\\String_",
            "type": {},
            "documentation": null
        }
    }
}